middlewares/isAdmin.js

```diff
@@ -0,0 +1,50 @@
+// midleware to check if the user.roleId user is admin
+const jwt = require('jsonwebtoken')
+const config = require('../config/config').development
+
+const isAdmin = (req, res, next) => {
+  const token = req.headers['x-access-token']
+  if (!token) {
+    return res.status(401).send({ auth: false, message: 'No token provided.' })
+  }
+
+  jwt.verify(token, config.secret, (err, decoded) => {
+    if (err) {
+      return res
+        .status(500)
+        .send({ auth: false, message: 'Failed to authenticate token.' })
+    }
+    if (decoded.roleId === 1) {
+      next()
+    } else {
+      return res
+        .status(403)
+        .send({ auth: false, message: 'You are not an admin' })
+    }
+    return null
+  })
+  return null
+}
+module.exports = isAdmin
+
+// examples
```

Comment:
instead of this commented code... you could do : 
a document of this is README.md